package.json

```diff
@@ -166,7 +166,6 @@
     "@qnighy/marshal": "0.1.3",
     "@renovatebot/detect-tools": "1.1.0",
     "@renovatebot/kbpgp": "4.0.1",
-    "@renovatebot/osv-offline": "1.5.12",
     "@mintmaker/osv-offline": "1.0.0",
     "@mintmaker/osv-offline-db": "1.0.0",
```

Comment:
Seems `osv-offline-db` can also be removed, I don't think we're calling any APIs from it directly in Renovate. @querti is that correct?